interp/interp_test.go

```diff
@@ -882,6 +889,10 @@ var runTests = []runTest{
 		`w="$HOME"; cd; [[ $PWD == "$w" ]]`,
 		"",
 	},
+	{
+		"cd /; cd /tmp; cd ''; echo $? $PWD $OLDPWD",
```

Comment:
this might fail as it depends on /tmp being present, and the tests also run on e.g. mac and windows. perhaps mkdir some directories and use those instead.

Reply:
Ah, thanks. It passes on my Macbook and under git-bash on Windows, but does indeed fail on Windows when not under the emulation layer. Fixed.